todoapp/app/src/main/java/com/example/android/architecture/blueprints/todoapp/addedittask/AddEditTaskPresenter.java

```diff
@@ -112,6 +116,11 @@ public void onDataNotAvailable() {
         }
     }
 
+    @Override
+    public boolean isDataMissing() {
```

Comment:
this is related to above, exposing business logic through the `Presenter` seems like a bad a idea